desktop/components/main_layout/templates/index.jade

```diff
@@ -1,6 +1,6 @@
 //- Override any locals with `append locals`
 block locals
-  - bodyClass = helpers ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
+  - bodyClass = helpers && helpers.buildBodyClass ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
```

Comment:
`main_layout` gets called from multiple places and assumes that method exists on `helpers`... in some places we pass `helpers` into a template, but in others we use a namespaced `ViewHelpers`. I just chose `helpers`.